packages/sanity/src/_internal/cli/commands/index.ts

```diff
@@ -110,6 +111,7 @@ const commands: (CliCommandDefinition | CliCommandGroupDefinition)[] = [
   previewCommand,
   uninstallCommand,
   execCommand,
+  extractManifestCommand,
```

Comment:
I believe we'll include a separate `CliCommandGroupDefinition` (e.g. extractManifestCommand) here as well

E.g.
```
// packages/sanity/src/_internal/cli/commands/manifest/extractManifestGroup.ts
import {type CliCommandGroupDefinition} from '@sanity/cli'

const extractManifestGroup: CliCommandGroupDefinition = {
  name: 'manifest',
  signature: '[COMMAND]',
  isGroupRoot: true,
  description: 'Extracts the studio configuration as one more JSON manifest files.',
}

export default extractManifestGroup
```

Without this, the manifest command won't appear in the list when running `sanity --help`, and running `sanity manifest` standalone will yield `Error: Command "manifest" is not available outside of a Sanity project context` (even if you are in a project context)